### ISSUE TITLE
Hold description not set?

### ISSUE THREAD

**Issue by mpolakis** [balanced/balanced-ruby#121]
I'm doing something like : 

``` ruby
hold = buyer.hold(
        amount: 34$,
        description: "yy yy yy",
        meta: { a: 5, b: 6, .. }
      )
```

Amount and meta attributes are set, while description is not.

Also, when voiding a hold, I can't pass the "appears_on_statement_as" parameter : ArgumentError: wrong number of arguments (1 for 0)

Please fix and edit your documentation.

**Comment by steveklabnik** [comment]
Apparently @peterkitchit is having this issue too.

@remear , have any ideas?

**Comment by steveklabnik** [comment]
01:53 < peter_kitchit> holds created by .debit have description, but not holds created by .hold

**Comment by ghost** [comment]
Seems to still be an issue, at least in 0.7.4

**Comment by remear** [comment]
Can you find a log in your Dashboard where you created a hold and supplied a description and see if it shows up in the payload. If it doesn't, there might be an issue in balanced-ruby. If it's there, then either there's an API issue, or `description` is not supported on holds.

**Comment by ghost** [comment]
I just check the logs, it is not in the request body. May be an issue with balanced-ruby. I'm on 0.7.5.

**Comment by remear** [comment]
@peterkitchit Would you please link me to the log.

**Comment by ghost** [comment]
@remear https://dashboard.balancedpayments.com/#/marketplaces/TEST-MP1d59tohwSRZnBesxXQ2zDw/logs/OHM894e78f2a8a711e39751026ba7d31e6f

**Comment by ghost** [comment]
Yeah, I just checked out the 0.8.0 branch. Hold descriptions are not set in the ruby gem.

Balanced::Account#hold does not include the code to set descriptions.

Could we get this fixed. It is quite inconvenient for us.

**Comment by mjallday** [comment]
@peterkitchit we've released [0.8.1](https://github.com/balanced/balanced-ruby/tree/v0.8.1) which should address this issue. You can view the changes on this commit - https://github.com/balanced/balanced-ruby/commit/363007432dd99bcaa124efaeef6f19d7f6a86894

We'll upload to ruby-gems shortly.
